meinberlin/assets/scss/components_user_facing/_live_questions.scss

```diff
@@ -7,3 +7,29 @@
     margin-left: auto;
     margin-bottom: .625em;
 }
+
+.live_question__progress {
+    display: flex;
+    gap: 8px;
+    margin-bottom: 8px;
+    position: relative;
+}
+
+.live_question__progress-bar {
+    background-color: $gray-lighter;
+    border-radius: 10px;
+    height: 100%;
+    position: absolute;
+    z-index: 0;
+}
+
+.live_question__progress-text {
+    padding-left: 8px;
+    padding-right: 8px; 
+    position: relative;
+    z-index: 1;
+}
+
+.live_question__progress-percentage {
+    font-weight: bold;
```

Comment:
Can you use `live-question` instead of `live_question`? Is more BEM-like and already defined like that in the rest of the file.